mysql-test/t/percona_bug1686603.test

```diff
@@ -0,0 +1,27 @@
+--source include/count_sessions.inc
+
+CREATE DATABASE dump;
+CREATE USER 'dumper'@'localhost';
+GRANT ALL ON dump.* TO 'dumper'@'localhost';
+
+CREATE TABLE dump.t(a int primary key auto_increment);
+INSERT INTO dump.t VALUES(NULL), (NULL), (NULL), (NULL);
+
+--connect (conn1,localhost,dumper,,dump)
+--connection conn1
+SELECT COUNT(*) FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA='performance_schema' AND TABLE_NAME='session_variables';
+--connection default
+--disconnect conn1
+
+--exec $MYSQL_DUMP dump t > $MYSQLTEST_VARDIR/tmp/t.sql
```

Comment:
I would use a more test-case-specific name here, like `percona_bug1686603_t.sql`.
`t.sql` seems to be very common, may have collisions with other tests.